ci/scripts/model_health_check.py

```
@@ -0,0 +1,349 @@
+from pathlib import Path
+
+try:
+    import yaml
+except ImportError:
+    print("ERROR: pyyaml is required. Install with: pip install pyyaml", file=sys.stderr)
+    sys.exit(1)
+
+try:
+    from gitutils import GitWrapper
+    _FALLBACK_REPO = GitWrapper.get_repo_dir()
+except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)